README.md

```diff
@@ -115,6 +116,20 @@ A multithreaded application server for PHP, written in PHP.
 
 - GitHub: [asyncphp](https://github.com/asyncphp)
 
+### Kraken
+
+PHP framework for creating multi-processed, multi-threaded, fault-tolerant applications using event-driven, message-driven and agent-based architecture.
+
+- GitHub: [kraken-php](https://github.com/kraken-php)
+- Website: [kraken-php.com](http://kraken-php.com)
+
+### phpDaemon
+
+Asynchronous server-side framework for network applications implemented in PHP using libevent.
+
+- GitHub: [phpdaemon](https://github.com/kakserpom/phpdaemon)
+- Website: [phpdaemon.io](https://daemon.io)
```

Comment:
phpdaemon.io does not resolve for me. If I go to daemon.io, I get an error: `NET::ERR_CERT_DATE_INVALID`.

Reply:
https://daemon.io/ works just fine

Reply:
@sevir Mind updating the web site link in this PR to reflect the correct hostname?